test/specs/undercover-init.el

```diff
@@ -1,7 +1,5 @@
 (when (require 'undercover nil t)
   (undercover "*.el"
-              (:exclude "*test*.el")
```

Comment:
I think undercover does not search subdirectories so this does not seem to be necessary.